api/api/settings.py

```diff
@@ -74,7 +76,10 @@
         "PASSWORD": env("POSTGRES_PASSWORD"),
         "HOST": env("POSTGRES_HOST"),
         "PORT": env("POSTGRES_PORT"),
-    }
+        "TEST": {
+            "NAME": env("POSTGRES_DB"),
+        },
+    },
```

Comment:
**question:** What is this for?

Reply:
without declaring this lines, tests run by default on the test database: "test_postgres", where pgvector extension is missing. I don't know how to fix this, so i temporarily run tests in the same database (i know this is incorrect), where extension is available.

![image](https://github.com/knmlprz/ChatKNML/assets/146839542/8d012063-23e2-410c-b856-922798bc51f7)